beacon_node/beacon_chain/src/beacon_chain.rs

```
@@ -668,6 +670,34 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         Ok(())
     }
 
+    /// Persists the custody information to disk.
+    pub fn persist_custody_context(&self) -> Result<(), Error> {
+        let custody_context: CustodyContextSsz = self
+            .data_availability_checker
+            .custody_context()
+            .as_ref()
+            .into();
+        debug!(?custody_context, "Persisting custody context to store");
+
+        if let Err(e) =
+            clear_custody_context::<T::EthSpec, T::HotStore, T::ColdStore>(self.store.clone())
+        {
+            error!(error = ?e, "Failed to clear old custody context");
+        }
+
+        match persist_custody_context::<T::EthSpec, T::HotStore, T::ColdStore>(
+            self.store.clone(),
+            custody_context,
+        ) {
+            Ok(_) => info!("Saved custody state"),
+            Err(e) => error!(
```

I think we want to return the error here? or is this intentional?

---

Doesn't matter. Same output.